sw/airborne/firmwares/rotorcraft/navigation.c

```diff
@@ -675,3 +676,24 @@ void nav_oval(uint8_t p1, uint8_t p2, float radius)
       return;
   }
 }
+
+#ifdef TRAFFIC_INFO
+#include "modules/multi/traffic_info.h"
+
+void nav_follow(uint8_t ac_id, uint32_t distance, uint32_t height)
+{
+  /*  struct EnuCoor_i* target = acInfoGetPositionEnu_i(ac_id);
```

Comment:
why is this commented out?
Still some fundamental problem here or simply not yet finished/tested?